news/2018-06-22-cwc-2018-recap-round-of-16.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "osu!catch World Cup 2018: Round of 16 Recap"
+date: 2018-06-22 12:00:00 +0000
+---
+
+Two weeks of frantic fruit-catching in the 2018 osu!catch World Cup completed, and several more still to look forward to - but who triumphed at the top of the tree, and who fell off the platter of progress in the Round of 16? Read on to find out more!
+
+![](/wiki/shared/news/banners/CWC_2018_banner.jpg)
+
+Hello once again one and all, [JBHyperion](https://osu.ppy.sh/users/4879508) here to deliver your weekly round-up on all the action from this year’s edition of the osu!catch World Cup. Following on from one of the most incredible Group Stages ever seen in an osu! World Cup, last weekend’s Round of 16 saw our remaining teams each jostling for position in their march to becoming this year’s *osu!catch Champions*.
```

Comment:
`’` -> `'`
The apostrophe is not the correct character unicode. Correct the other instances.

Reply:
Imagine still having to fix these in 2018... Sigh, '-ified al